POSIX/posix-abstract-declarator.c

```diff
@@ -0,0 +1,25 @@
+/* To trigger the problem, we must not have an argument name.
```

Comment:
`/**`

Also, do you want to require `-DARGNAME=""` vs. `-DARGNAME="var"`?  I was thinking that we could `#define ARGNAME` based on whether POSIXFAIL_ABSTRACT_DECLARATOR was defined.

Reply:
Hmm... none of the other `POSIX/*.c` files make use of `POSIXFAIL_*`, but that _would_ simplify `posix-cflags.sh`.  Sure.